charts/ccd-case-management-web/values.template.yaml

```diff
@@ -13,3 +13,10 @@ nodejs:
     CCD_ACTIVITY_RETRY: 5
     CCD_ACTIVITY_BATCH_COLLECTION_DELAY_MS: 1
     CCD_ACTIVITY_MAX_REQUEST_PER_BATCH: 25
+
```

Comment:
this should be applied to preview env only. Since we don't have a dedicated file for preview, wonder whether this will impact other envs as well

Reply:
ideally yes :). I guess we also need to change values.template.yaml to values.yaml as there is Jenkins warning atm. 
Eg: https://build.platform.hmcts.net/job/HMCTS_CDM/job/ccd-case-management-web/view/change-requests/job/PR-722/3/consoleFull
search for "values.yaml".  I'll change both of them